x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/mocks/state.ts

```diff
@@ -420,6 +420,7 @@ export const mockState: State = {
     dataStreamDescription: 'Mocked Data Stream Description',
     inputTypes: ['filestream'],
     logsSampleParsed: rawSamples,
+    samplesFormat: { name: 'ndjson', multiline: false },
```

Comment:
I see that here `logsSampleFormat` would make sense. 

But I feel like we should decide – do we have one sample? Or many samples? Is it necessarily a _log_ sample? Currently it's 

`logs*Sample*Parsed: raw*Samples*`.

Reply:
It sounds like `samples` is more widespread at this point. I can just rename `logsSampleParsed` which is frontend-only internal state.

Reply:
This state is going to change in the next PR when I move the log parsing logic to the server. So can leave it for now